_draft/2017-07-31-why-jwt.md

```diff
@@ -0,0 +1,52 @@
+---
+title:  "Why JWT?"
+author: iviczl
+image: '/img/nuget-highway.jpg'
+tags: [jwt, token, authentication]
+---
+# Authentication Exposure
+
+However sensenet ECM has been supporting authentication through NTLM and Basic for a long time, neither of them is a good choice for some scenarios. NTLM can be used only, if the authenticating user is a Windows domain user as well, Basic authentication on the other hand requires to send your user's credentials with every request. 
+We decided to implement a modern and easily usable and secure enough means to authenticate a user in the sensenet ECM environment.
+
+### Why JWT?
+There were a plethora of authentication methods in the wild, but we needed only one. 
```

Comment:
This felt wrong, but apparently it can be right. :) See here: https://english.stackexchange.com/questions/103176/there-is-a-plethora-or-there-are-a-plethora

It's worth the read. Going with the second commenter's logic, this sohuld be "there was a plethora of [...]". But this is arguable.